iib/workers/tasks/utils.py

```diff
@@ -723,13 +723,23 @@ def run_cmd(cmd, params=None, exc_msg=None, strict=True):
     if strict and response.returncode != 0:
         log.error('The command "%s" failed with: %s', ' '.join(cmd), response.stderr)
         if cmd[0] == 'opm':
-            # Capture the error message right before the help display
-            regex = r'^(?:Error: )(.+)$'
-            # Start from the last log message since the failure occurs near the bottom
-            for msg in reversed(response.stderr.splitlines()):
-                match = re.match(regex, msg)
-                if match:
-                    raise IIBError(f'{exc_msg.rstrip(".")}: {match.groups()[0]}')
+            # The new OPM shows the error at the beginning, let's try with this first
+            regex = r'^(?:Error: )(.+)'
+            msg = response.stderr.splitlines()
+            match = re.match(regex, "".join(msg))
+            if match:
+                raise IIBError(f'{exc_msg.rstrip(".")}: {match.groups()[0]}')
+            else:
```

Comment:
nitpick - This `else` is not necessary, you can save indentation.

Reply:
Thanks! I'll change it.

Reply:
I've sent the amended commit